src/applications/vre/28-1900/components/VeteranInformationViewComponent.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect } from 'react';
+import merge from 'lodash/merge';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import Telephone, {
+  CONTACTS,
+} from '@department-of-veterans-affairs/formation-react/Telephone';
+import moment from 'moment';
+
+const NAME_PATH = 'veteranInformation.fullName';
+const DOB_PATH = 'veteranInformation.dob';
+
+export const VeteranInformationViewComponent = props => {
+  const { profile, formData, setData } = props;
+  const {
+    gender,
+    dob,
+    userFullName: { first, last },
+  } = profile;
+  let dateOfBirthFormatted = '-';
+  let genderFull = '-';
+  if (dob) {
+    dateOfBirthFormatted = moment(dob).format('MMMM Do YYYY');
+  }
+  if (gender === 'M') {
+    genderFull = 'Male';
+  } else if (gender === 'F') {
+    genderFull = 'Female';
+  }
+
+  // Update the formData with values pulled from profile
+  const veteranFormData = {
+    veteranInformation: {
+      fullName: profile?.userFullName,
+      dob: profile?.dob,
+    },
+    veteranAddress: {},
+  };
+  const updatedFormData = { ...formData, ...veteranFormData };
+  useEffect(
+    () => {
+      setData(updatedFormData);
```

Comment:
If a user is logged in, we need to update the form data with values given to us from the profile.